.github/workflows/build-artifacts.yml

```diff
@@ -0,0 +1,36 @@
+name: Build artifacts
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - src/**
+
+jobs:
+  pack:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use .NET Core 3.1 SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.x'
+      - name: Install dependencies
+        working-directory: src
+        run: dotnet restore
+      - name: Build solution (release config)
+        working-directory: src
+        run: dotnet build --no-restore -c Release -p:NoWarn=CS1591 -p:VersionSuffix=${{ github.run_id }}
+      - name: Pack solution (release config)
+        working-directory: src
+        run: dotnet pack --no-restore -c Release -p:VersionSuffix=${{ github.run_id }} -o out
+      - name: Publish artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: Nuget packages
+          path: |
+            src/out/*.nupkg
+#      - name: Publish Nuget packages to GitHub registry
```

Comment:
GitHub registry or MyGet?

Reply:
After much testing and investigation, I’ve determined that the GitHub registry is only accessible to authenticated users, so probably continue using MyGet. We should update the comment if/when we start using this in place of appveyor.

Note that I was unable to get it to upload to the GitHub registry with the GITHUB_TOKEN as shown in the documentation and multiple websites. It would only work with a personal access token. I was probably doing something wrong but I couldn’t figure it out. 

Second, the unique identifier is 8 digits or so instead of the nice short 4 digit ones created by appveyor.

However, the integration with GitHub for artifacts, in progress jobs and build errors is rather nice. It also runs notably faster than appveyor.

Reply:
I should probably get it working to the GitHub registry ... just to try it out. Maybe GitHub will allow public access at some point. We can always upload to both myget and GitHub packages at once rather easily. 

I just need to create a GitHub account with the proper privileges.